packages/manager/modules/pci/src/projects/project/training/jobs/info/info.html

```diff
@@ -279,8 +279,10 @@
                                 ng-repeat="data in $ctrl.job.spec.volumes track by $index"
                                 class="api-list-item"
                             >
-                                {{ data.container }}@{{
-                                data.region}}:{{data.mountPath}}:{{data.permission}}
+                                {{ data.container }}@{{data.region}}<span
+                                    ng-if="data.prefix !== ''"
+                                    >/{{data.prefix}}</span
+                                >:{{data.mountPath}}:{{data.permission}}
```

Comment:
i think we can have something like that:

```
getVolumeText(volume) { 
   const prefix = volume.prefix ? volume.prefix : '';
   return `${volume.container}@${volume.region}${prefix}:${volume.mounthPath}`;
}
```

and use it:
```suggestion
      <span data-ng-bind="$ctrl.getVolumeText(volume)">/span>
```

Reply:
Done, thanks for the suggestion